### PR TITLE
SGD8-601: Add figcaption tag around all field_media_caption fields

### PR DESCRIPTION
**Pull request by hstandaert**
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.

### CODE REVIEW

File: templates/custom/media/media--image--gallery-multiple.html.twig [hunk 1]
```diff
@@ -28,8 +28,7 @@
     <div class="image-wrapper" data-ratio="8:5">
       {{ content.field_media_image }}
     </div>
-    {% if content.field_media_caption|render|trim is not empty %}
-      <figcaption>{{ content.field_media_caption }}</figcaption>
-    {% endif %}
+
+    {{ content.field_media_caption }}
```
Comment (meirege) [inline templates/custom/media/media--image--gallery-multiple.html.twig:32]: No check necessary?

File: templates/custom/media/media--image--gallery-single.html.twig [hunk 2]
```diff
@@ -26,8 +26,6 @@
   {{ title_suffix.contextual_links }}
   <figure>
     {{ content.field_media_image }}
-    {% if content.field_media_caption|render|trim is not empty %}
-      <figcaption>{{ content.field_media_caption }}</figcaption>
-    {% endif %}
+    {{ content.field_media_caption }}
```
Comment (meirege) [inline templates/custom/media/media--image--gallery-single.html.twig:29]: No check necessary?

File: templates/custom/media/media--wide-image.html.twig [hunk 3]
```diff
@@ -14,6 +14,5 @@
 #}
 
 {% if content %}
-  {{ content|without('field_media_caption') }}
-  <figcaption>{{ content.field_media_caption }}</figcaption>
+  {{ content }}
```
Comment (meirege) [inline templates/custom/media/media--wide-image.html.twig:17]: Same as above? :)

File: templates/custom/media/media--image--gallery-multiple.html.twig (hunk 1 above)
Comment (hstandaert) [reply templates/custom/media/media--image--gallery-multiple.html.twig:32]: Drupal doesn't render anything when it's empty, right? The extra checks are only necessary when extra markup is added in the template.

**Review by meirege — APPROVED** [review]

**Pull request merged**